### DISCUSSION TITLE
Mixer tab

### DISCUSSION THREAD

**Discussion by Vixamacoin** [iNavFlight/inav#10347] [Ideas]
Good evening, everyone.

I’m reaching out because I need some assistance. I’ve recently moved from BetaFlight to Inav, and I’m running into some issues. Specifically, I’m having trouble with the mixer settings and have discovered three motors connected to the timer’s output.

Any help or guidance you can provide would be greatly appreciated!

Thank you in advance.
![20240909_235941](https://github.com/user-attachments/assets/6f41429e-cde5-4c3a-9c90-29fa2f258bad)
![20240909_234912](https://github.com/user-attachments/assets/bb7290f6-783f-4d06-91f7-56ed62bb721f)

**Comment by sensei-hacker** [comment]
What exactly is your question, or the problem you are trying to solve?
I understand you have used Betaflight before, and you have some question regarding the mixer?
